js/popover.js

```diff
@@ -41,14 +41,10 @@
   }
 
   Popover.prototype.setContent = function () {
-    var $tip    = this.tip()
-    var title   = this.getTitle()
-    var content = this.getContent()
-
-    $tip.find('.popover-title')[this.options.html ? 'html' : 'text'](title)
-    $tip.find('.popover-content').children().detach().end()[ // we use append for html objects to maintain js events
```

Comment:
@cvrebert Right, I've added `.empty()` before append in the case where an element is being appended. I don't think we should `.detach()` as it is almost never useful but always creates memory leaks when it's not. The difference between `empty` and `detach` is that the latter keeps all the attached jQuery events and data, but since we do not return the references to previous children the user would have to keep these references themselves, thus they can `detach` themselves anyway if they need to.